openapi/kas-fleet-manager.yaml

```diff
@@ -256,6 +256,81 @@ paths:
                   $ref: '#/components/examples/500Example'
     parameters:
       - $ref: "#/components/parameters/id"
+  /api/kafkas_mgmt/v1/kafkas/{id}/promote:
+    parameters:
+      - $ref: "#/components/parameters/id"
```

Comment:
@miguelsorianod @JameelB @ziccardi in some endpoints e.g https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0c9948ab90b1e1eb45b032402b6a6b948f67ddf3/openapi/kas-fleet-manager.yaml#L338-L339 there is a usage of `async=true` to indicate that the operation is going to happen in async way. Is this something that can be used here as well?

Reply:
I am aware of that and I also thought about it.
This endpoint is going to be asynchronous and there won't be a way to do it in a synchronous way. Is there any particular reason where to add the `async` query parameter? for consistency with the other endpoints? To make explicit the asynchronous nature of the endpoint? In any case, even with adding the parameter the endpoint documentation should document that the endpoint is asynchronous (as it is already done)

cc @ziccardi @JameelB for other views on this

Reply:
> Is there any particular reason where to add the async query parameter? for consistency with the other endpoints? To make explicit the asynchronous nature of the endpoint? 

+1 on these two points. I think all async endpoints have this, so it'll be consistency with other endpoints. And the way this query parameter was mainly used (from my understanding) is to make it explicit the async nature of the endpoint.

Reply:
Agree with @machi1990 about adding it for consistency.

Reply:
added

Reply:
Thanks @miguelsorianod @ziccardi